web/docs/deployment/database.md

```diff
@@ -2,10 +2,66 @@
+
+The benefit of migrations is that you can apply the same change to multiple databases, e.g. your local development database and your production database. 
+
+### Creating migrations
+
+After you made a change in the Prisma schema, you can create a migration by running the following command:
+
+```bash
+wasp db migrate-dev
+```
+
+This command will create a new migration in the `migrations` directory. The migration is a set of SQL commands that describe the change you made in the schema.
+
+### Applying migrations
+
+**In development**, the migrations are applied as soon as you run the `wasp start` command. 
+
+**In production**, the server app first checks if there are any new migrations that need to be applied, and if there are, it applies them before starting the server. This way, the database schema is always in sync with the Prisma schema.
+
+The migrations might fail to apply if there is a conflict with the existing data in the database. In that case, you'll need to fix the migration and try again.
+
+### Debugging failed migrations
+
+If a migration fails to apply, the server app will log the error message and stop. You should then connect to the production database and see what went wrong. If you check the `_prisma_migrations` table, you'll see the failed migration there. 
+
+You can try resolving the erorr e.g. if you tried adding a `@unique` constraint to a field that already has duplicate values:
+1. Remove any duplicate values from the database
+2. Remove the failed migration from the `_prisma_migrations` table
+3. Try applying the migration again by restarting the server app
+
+:::tip Viewing the `_prisma_migrations` table
+
+You can't use the `wasp db studio` command to view the `_prisma_migrations` table in the production database, but you can use a database management tool like [DBeaver](https://dbeaver.io/) or [pgAdmin](https://www.pgadmin.org/).
+:::
+
+## Connect to the production database
+
+**In development**, you can use the `wasp db studio` command to open a web-based database management tool that allows you to inspect the database. 
+
+You can use the same tool to inspect the **production database**, but you'll need to set the `DATABASE_URL` environment variable to point to the production database. Set the variable in the `.env.server` file in the root of your project.
+
+```bash title=".env.server"
+DATABASE_URL="postgresql://user:password@host:port/dbname"
+```
+
+Then you can start the database studio with the following command:
+
+```bash
+wasp db studio
+```
+
+If you are looking how to connect to a Fly.io production database, we wrote a [guide](https://github.com/wasp-lang/learning-materials/?tab=readme-ov-file#running-wasp-db-studio-on-production-db) on how to do that.
```

Comment:
This is quite dangerous though, they can easily forget to unset that value and then in development to stuff on production database. So if they are doing this, I would super strongly emphasize to unset it as soon as they are done. That said, I would prefer suggesting different approach. For example setting it in shell? That way it stays only for the current session. They could even use it only per command, as prefix `DATABASE_URL=<production_url> wasp db studio`, I think that would work? Maybe they can create .sh file for it also (although they have to be careful not to commit it). Maybe safest is just to tell them to set env var in session.

Reply:
Good stuff!